### DISCUSSION TITLE
Fixing knownStates with fitCTHMM

### DISCUSSION THREAD

**Discussion by chrisberglund** [bmcclintock/momentuHMM#185] [Q&A]
Hello,

I've got telemetry data from some seals, and I'm looking at trying the multistate Langevin diffusion from the recent paper using fitCTHMM. However, what I'd like to do is to have the transition rate in and out of the hauled out state to only depend on information from the wet/dry sensor. With the discrete-time HMM, I'd fix the beta values along with providing the knownStates, but for that to work requires the betaRef parameter which fitCTHMM does not have. Is there a different way to do something similar with the continuous time HMM or is there something I'm missing that makes it not a good idea?

**Comment by bmcclintock** [comment]
Could you provide more information on exactly what you’re trying to do? Preferably a more detailed description including equations and/or code. The betaRef argument is not included in fitCTHMM owing to constraints imposed on the transition rate matrix, but with more detail I could try and help.

**Comment by chrisberglund** [comment]
Thank you for the reply, and thank you so much for your work on this package. It is a really fantastic package. With a bit more tinkering, I managed to get good fits just using wet/dry has a data stream in addition to position. So I'm not going to have to do what I was originally thinking of doing, but it is good to know about the constraints imposed by the transition rate matrix.
